server/clearbit/enrich.js

```diff
@@ -70,7 +68,11 @@ function fetchFromReveal(user = {}, clearbit) {
   return clearbit.client
     .reveal({ ip })
     .then(({ company }) => {
+      console.warn("Revealed ip : ", JSON.stringify({ ip, company }));
```

Comment:
maybe we can use `ctx.client.logger` instead of `console.log`.
We will have the org/ship context thanks to that, and when you set the level to debug it won't be shown in customer faced logs.